### PR TITLE
SOFTWARE-3670 Failures on TPC testing

### PR DESCRIPTION
**Pull request by efajardo**

### CODE REVIEW

File: osgtest/tests/test_465_xrootd_tpc.py [hunk 1]
```diff
@@ -10,8 +10,10 @@
 class TestXrootdTPC(osgunittest.OSGTestCase):
 
     def test_01_create_macaroons(self):
-        core.skip_ok_unless_installed('xrootd', 'xrootd-scitokens', 'x509-scitokens-issuer-client', by_dependency=True)
+        core.skip_ok_unless_installed('xrootd', 'x509-scitokens-issuer-client', by_dependency=True)
         self.skip_bad_unless(core.state['proxy.created'], 'Proxy creation failed')
+        core.config['xrootd.tpc.macaroon-1'] = None;
```
Comment (matyasselmeci) [inline osgtest/tests/test_465_xrootd_tpc.py]:
```suggestion
        core.config['xrootd.tpc.macaroon-1'] = None
```

File: osgtest/tests/test_465_xrootd_tpc.py [hunk 2]
```diff
@@ -10,8 +10,10 @@
...
+        core.config['xrootd.tpc.macaroon-2'] = None;
```
Comment (matyasselmeci) [inline osgtest/tests/test_465_xrootd_tpc.py]:
```suggestion
        core.config['xrootd.tpc.macaroon-2'] = None
```

File: osgtest/tests/test_465_xrootd_tpc.py [hunk 3]
```diff
@@ -23,17 +25,21 @@ def test_01_create_macaroons(self):
...
         fail = core.diagnose('Obtain Macaroon number two',
                              command, status, stdout, stderr)
+        self.assertEqual(status, 0, fail)
         core.config['xrootd.tpc.macaroon-2'] = stdout.strip()
         
     def test_02_initate_tpc(self):
-        core.skip_ok_unless_installed('xrootd', 'xrootd-scitokens', 'x509-scitokens-issuer-client', by_dependency=True)
+        core.skip_ok_unless_installed('xrootd', 'x509-scitokens-issuer-client', by_dependency=True)
+        self.skip_bad_unless(core.config['xrootd.tpc.macaroon-1']!=None, 'Macaroon creation failed earlier')
+        self.skip_bad_unless(core.config['xrootd.tpc.macaroon-2']!=None, 'Macaroon creation failed earlier')
```
[the remark this answers is not in the document]
Comment (efajardo) [reply osgtest/tests/test_465_xrootd_tpc.py]: Yes I need the binary: `/usr/bin/macaroon-init`

File: osgtest/tests/test_465_xrootd_tpc.py (hunk 1 above)
Comment (efajardo) [reply osgtest/tests/test_465_xrootd_tpc.py]: Sorry too much c++ programming lately

**Comment by efajardo** [comment]
@matyasselmeci can you review again?

**Pull request merged**